### PR TITLE
Update get-started-windows-365-business.md

### PR DESCRIPTION
**Pull request by yuhonda**
This description lead to misunderstand that mobile device is not supported for Cloud PC. I suggest that add additional description about NOTE in Windows 365 home page section.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/admin/setup/get-started-windows-365-business.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ On their Windows 365 home page, users see the Cloud PCs they have access to in t
 Users can select **Open in browser** to open their Cloud PC.
 
 > [!NOTE]  
-> Mobile devices aren’t currently supported.
+> Mobile devices aren’t currently supported to use browser to open the Cloud PC. Remote Desktop app is supported.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/setup/get-started-windows-365-business.md]:
```suggestion
> Mobile devices aren’t currently supported for using a browser to open Cloud PC. The Remote Desktop app is supported.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @yuhonda, kindly commit my suggested edit. Many thanks.

**Comment by yuhonda** [comment]
Thank you I have commited.

**Comment by yuhonda** [comment]
Sorry. I missed to close PR. I have committed suggestion and please consider to merge.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@efrene Please merge this PR. Thanks

**Pull request merged**